cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -408,6 +416,13 @@ func main() {
 		log.SetOutput(ioutil.Discard)
 	}
 
+	if *fdRlimit == 0 {
+		*fdRlimit = expectedFDs
```

Comment:
What about making `expectedFDs` the default value for that flag?

Reply:
(and maybe when users set it to 0, we disable trying to touch the rlimit?)

Reply:
Done.